include/boost/geometry/algorithms/detail/overlay/get_turn_info_for_endpoint.hpp

```diff
@@ -197,7 +197,7 @@ class linear_intersections
     template <std::size_t I>
     ip_info const& get() const
     {
-        BOOST_STATIC_ASSERT(I < 2);
+        static_assert(I < 2, "I must not be greater than 1");
```

Comment:
This sounds like a funny sentence ;-)

Reply:
It is no problem, it just sounds like `I (me) should be too small` ;-) 
You can keep it.